client/components/jetpack/daily-backup-status/status-card/backup-in-progress.tsx

```diff
@@ -63,9 +62,9 @@ const BackupInProgress: React.FC< Props > = ( { percent, inProgressDate, lastBac
 			<ProgressBar value={ percent } total={ 100 } />
 
 			{ siteLastBackupDate && (
-				<div className="status-card__no-backup-last-backup">
+				<p className="status-card__no-backup-last-backup">
 					{ translate( 'Last backup before today: {{link}}%(lastBackupDisplayDate)s{{/link}}', {
-						args: { lastBackupDisplayDate: preventWidows( lastBackupDisplayDate ) },
+						args: { lastBackupDisplayDate: lastBackupDisplayDate || '' },
 						components: {
```

Comment:
Test on an atomic site, by going to Jetpack > backups and starting a backup. Shrink the screen until the affected elements wrap:

<img width="300" alt="Screenshot 2025-05-08 at 3 06 26 pm" src="https://github.com/user-attachments/assets/05d4841a-7475-4b46-91fe-7802ac1d9e7a" />